azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -343,14 +372,23 @@ def __init__(self, name: str, provider: "AzureQuantumProvider", **kwargs):
             "configuration", default_config
         )
         super().__init__(configuration=configuration, provider=provider, **kwargs)
+    
+    def _azure_config(self) -> Dict[str, str]:
+        config = super()._azure_config()
+        config.update(
+            {
+                "is_default": True,
```

Comment:
I'm not sure this is correct. Wouldn't all backends derived from `IonQBackend`should keep the default config from their parent class such that `is_default` is set to `False`?

Reply:
This change is due to the `is_passthrough` flag... we want to specify a default passthrough and a default QIR backend, and then in filtration of the backends, we can handle whether to use the QIR or passthrough default... this change was made because if not, then selection of ionq backends for example would require specification of both input format and gateset. With the `is_passthrough` flag, we can still maintain the same defaults for passthrough backends, while also having defaults for QIR backends. In filtration we then prefer QIR defaults > passthrough defaults